src/test/scala/uk/gov/nationalarchives/consignmentexport/KeycloakClientSpec.scala

```diff
@@ -1,13 +1,15 @@
 package uk.gov.nationalarchives.consignmentexport
 
-import uk.gov.nationalarchives.consignmentexport.Config.{Api, Auth, Configuration, EFS, S3}
+import uk.gov.nationalarchives.consignmentexport.Config.{Api, Auth, Configuration, EFS, S3, SFN}
```

Comment:
I know it doesn't matter in the slightest but can this be `uk.gov.nationalarchives.consignmentexport.Config._` like the other ones you've changed?